08LessonEight/homework/homework-08.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Home Work 08</title>
+    <link rel="stylesheet" href="./css/style.css"
+</head>
+
+<body>
+  <main>
+    <div class="box">
+      <img src="./img/truck.jpg">
+      <h6>SAFE & SECURE DELIVERY<h6>
+      <p>Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+      </p>
+      <br>
+      <div id="readmorebox">READ MORE</div>
+      <div id="redbox"></div>
+    </div>
+
+    <div class="box">
+      <img src="./img/plane.jpg">
+      <h6>SAFE & SECURE DELIVERY<h6>
+      <p>Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+      </p>
+      <br>
+      <div id="readmorebox">READ MORE</div>
+      <div id="redbox"></div>
```

Comment:
you should only have one element with the redbox id! if you want multiple elements with these properties, make redbox into a class instead!